Source/objects.cpp

```diff
@@ -1570,9 +1591,7 @@ void UpdateFlameTrap(int i)
 		if (Objects[i]._oVar4 != 0)
 			ActivateTrapLine(Objects[i]._otype, Objects[i]._oVar1);
 	} else {
-		int damage[4] = { 6, 8, 10, 12 };
-
-		int mindam = damage[leveltype - 1];
+		int mindam = 4 + leveltype * 2;

```

Comment:
```suggestion
		int mindam = 4 + static_cast<int>(leveltype) * 2;
```

Using an enum as a numeric value, might as well be explicit about the conversion. This also means that traps are doing more damage on hive/crypt than hellfire does but I assume that's an acceptable/intended change.

Reply:
This was a bit of a mistake on my part. Since i read this as meaning Crypt don't have traps, and forgot about the nest :sweat_smile: 
```cpp
		if (leveltype > DTYPE_CATHEDRAL)
			AddChestTraps();
```
But this only means that they are less frequent since  they can still be added by `AddObjTraps()`

Reply:
Since the burning crosses never spawn in the Nest or Crypt it is a bit up to our own interpretation how they should behave for custom levels. I did go ahead and revert this and instead mirror Cave and Hell since that is what the balance seams to mostly match.